lib/dynamoid/transaction/delete.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'action'
+
+module Dynamoid
+  class Transaction
+    class Delete < Action
+
+      def initialize(model_or_model_class, id = nil, options = {})
+        model = model_or_model_class.is_a?(Dynamoid::Document) ?
+                  model_or_model_class : model_or_model_class.find(id, consistent_read: true)
```

Comment:
**minor:** It seems loading model from the storage is not needed.

Reply:
Agreed. This is not needed for delete nor destroy.

Reply:
delete no longer instantiates.